docker-compose/ton-node/build/Dockerfile

```diff
@@ -1,5 +1,9 @@
+ARG RUST_VERSION=1.50.0
+
 FROM ubuntu:18.04 as build
 
+ARG RUST_VERSION
```

Comment:
Would it be enough just to keep this line `ARG RUST_VERSION=1.50.0` and remove 1st line?